card/lib/cardio.rb

```diff
@@ -50,26 +48,19 @@ def set_config config
   
     def set_paths paths
       @@paths = paths
-      paths.add 'tmp/lib'
-      paths.add 'tmp/set'
-      paths.add 'tmp/set_pattern'
-      paths.add 'db/migrate_deck_cards', :with=>'db/migrate_cards'
+      add_gem_path 'tmp/set', :root => root
+      add_gem_path 'tmp/set_pattern', :root => root
```

Comment:
That part I understood, the question is what did you want in the fix.  I think you just want the rename, not two methods, so that's what I'll do.

Actually, doesn't this method name pre-date the split?  Oh, you added the root option, right, then I abused it :)